apps/remix-ide/src/app/providers/environment-explorer.tsx

```diff
@@ -99,88 +99,100 @@ export class EnvironmentExplorer extends ViewPlugin {
       >
         <RemixUIGridSection
           plugin={this}
-          title='Injected'
-          hScrollable={true}
+          title='Deploy using a Browser Extension.'
+          hScrollable={false}
         >
           {this.providers['Injected'].map(provider => {
             return <RemixUIGridCell
               plugin={this}
-              title={provider.name}
+              title={provider.displayName}
+              logos={provider.logos}
               classList='EECellStyle'
+              payload={provider.description}
               pinned={this.pinnedProviders.includes(provider.name)}
               pinStateCallback={async (pinned: boolean) => {
                 if (pinned) {
                   this.emit('providerPinned', provider.name, provider)
+                  this.call('notification', 'toast', `"${provider.displayName}" has been added to the Network list of the Deploy & Run Transactions module.`)
```

Comment:
It should be 'added to the ~~Network~~ Environment list of the Deploy & Run Transactions ~module~ plugin'